js/lazy_images.js

```diff
@@ -17,6 +18,10 @@ function IsImageOk(img) {
   return true;
 }
 
+function arrayContains(needle, arrhaystack)
```

Comment:
I think this can be substitued with the array.includes() method: 
https://www.w3schools.com/Jsref/jsref_includes_array.asp